x-pack/plugins/watcher/__jest__/client_integration/watch_create_threshold.test.tsx

```diff
@@ -0,0 +1,818 @@
+  EuiComboBox: (props: any) => (
+    <input
+      data-test-subj="mockComboBox"
+      onChange={async (syntheticEvent: any) => {
+        props.onChange([syntheticEvent['0']]);
+      }}
+    />
+  ),
+}));
+
+const { setup } = pageHelpers.watchCreateThreshold;
+
+describe.skip('<ThresholdWatchEdit /> create route', () => {
+  const { server, httpRequestsMockHelpers } = setupEnvironment();
+  let testBed: WatchCreateThresholdTestBed;
+
+  afterAll(() => {
+    server.restore();
+  });
+
+  describe('on component mount', () => {
+    beforeEach(async () => {
+      testBed = await setup();
+
+      // @ts-ignore (remove when react 16.9.0 is released)
+      await act(async () => {
+        const { component } = testBed;
+        await nextTick();
+        component.update();
+      });
+    });
+
+    test('should set the correct page title', () => {
+      const { find } = testBed;
+
+      expect(find('pageTitle').text()).toBe('Create threshold alert');
+    });
+
+    describe('create', () => {
+      beforeEach(async () => {
+        httpRequestsMockHelpers.setLoadMatchingIndicesResponse({ indices: MATCH_INDICES });
+        httpRequestsMockHelpers.setLoadEsFieldsResponse({ fields: ES_FIELDS });
+        httpRequestsMockHelpers.setLoadSettingsResponse(SETTINGS);
+        httpRequestsMockHelpers.setLoadWatchVisualizeResponse(WATCH_VISUALIZE_DATA);
+
+        testBed = await setup();
+
+        // @ts-ignore (remove when react 16.9.0 is released)
+        await act(async () => {
+          await nextTick();
+          testBed.component.update();
+        });
+      });
+
+      describe('form validation', () => {
+        test('should not allow empty name field', () => {
+          const { form } = testBed;
+
+          form.setInputValue('nameInput', '');
+
+          expect(form.getErrorsMessages()).toContain('Name is required');
+        });
+
+        test('should not allow empty time field', () => {
+          const { form } = testBed;
+
+          form.setInputValue('watchTimeFieldSelect', '');
+
+          expect(form.getErrorsMessages()).toContain('A time field is required');
+        });
+
+        test('should not allow empty interval size field', () => {
+          const { form } = testBed;
+
+          form.setInputValue('triggerIntervalSizeInput', '');
+
+          expect(form.getErrorsMessages()).toContain('Interval size is required');
+        });
+
+        test('should not allow negative interval size field', () => {
+          const { form } = testBed;
+
+          form.setInputValue('triggerIntervalSizeInput', '-1');
+
+          expect(form.getErrorsMessages()).toContain('Interval size cannot be a negative number');
+        });
+
+        test('should disable the Create button with invalid fields', () => {
+          const { find } = testBed;
+
+          expect(
+            find('saveWatchButton')
+              .getDOMNode()
+              .getAttribute('disabled')
```

Comment:
Doesn't `find('saveWatchButton').props().disabled` give us a boolean to test?

Reply:
good point, fixed!